### PR TITLE
[enterprise-3.10] bug 1609697 update disconnected install

### PR DESCRIPTION
**Pull request by kalexand-rh**
From https://github.com/openshift/openshift-docs/pull/12024

### CODE REVIEW

File: install/disconnected_install.adoc [hunk 1]
```diff
@@ -578,247 +610,44 @@ name=rhel-7-server-ansible-2.4-rpms
 baseurl=http://<server_IP>/repos/rhel-7-server-ansible-2.4-rpms <1>
 enabled=1
 gpgcheck=0
-[rhel-7-server-ose-3.10-rpms]
-name=rhel-7-server-ose-3.10-rpms
-baseurl=http://<server_IP>/repos/rhel-7-server-ose-3.10-rpms <1>
+[rhel-7-server-ose-3.11-rpms]
```
Comment (jianlinliu) [inline install/disconnected_install.adoc]: This not right for 3.10.

File: install/disconnected_install.adoc [hunk 2]
```diff
@@ -330,72 +313,68 @@ For example, to pull both the previous and latest version of the Tomcat image:
...
+$ docker pull registry.access.redhat.com/rhscl/perl-524-rhel7:<tag>
+$ docker pull registry.access.redhat.com/rhscl/php-56-rhel7:<tag>
+$ docker pull registry.access.redhat.com/rhscl/postgresql-95-rhel7:<tag>
+$ docker pull registry.access.redhat.com/rhscl/python-35-rhel7:<tag>
+$ docker pull registry.access.redhat.com/redhat-sso-7/sso70-openshift:<tag>
+$ docker pull registry.access.redhat.com/rhscl/ruby-24-rhel7:<tag>
+$ docker pull registry.access.redhat.com/redhat-openjdk-18/openjdk18-openshift:<tag>
+$ docker pull registry.access.redhat.com/redhat-sso-7/sso71-openshift:<tag>
+$ docker pull registry.access.redhat.com/rhscl/nodejs-6-rhel7:<tag>
+$ docker pull registry.access.redhat.com/rhscl/mariadb-101-rhel7:<tag>
```
Comment (jianlinliu) [inline install/disconnected_install.adoc]: I do not think those s2i images which is not under openshift3 are following the same tag version like openshift version, e.g: the present openshfit version is 3.10.45, rhscl/nodejs-6-rhel7 has no such version tag.

File: install/disconnected_install.adoc [hunk 3]
```diff
@@ -274,6 +267,17 @@ $ docker pull registry.access.redhat.com/rhgs3/rhgs-server-rhel7
...
+$ docker pull registry.access.redhat.com/cloudforms46/cfme-openshift-memcached:<tag>
+$ docker pull registry.access.redhat.com/cloudforms46/cfme-openshift-app-ui:<tag>
+$ docker pull registry.access.redhat.com/cloudforms46/cfme-openshift-app:<tag>
+$ docker pull registry.access.redhat.com/cloudforms46/cfme-openshift-embedded-ansible:<tag>
+$ docker pull registry.access.redhat.com/cloudforms46/cfme-openshift-httpd:<tag>
+$ docker pull registry.access.redhat.com/cloudforms46/cfme-httpd-configmap-generator:<tag>
+$ docker pull registry.access.redhat.com/rhgs3/rhgs-server-rhel7:<tag>
+$ docker pull registry.access.redhat.com/rhgs3/rhgs-volmanager-rhel7:<tag>
+$ docker pull registry.access.redhat.com/rhgs3/rhgs-gluster-block-prov-rhel7:<tag>
+$ docker pull registry.access.redhat.com/rhgs3/rhgs-s3-server-rhel7:<tag>
```
Comment (jianlinliu) [inline install/disconnected_install.adoc]: The same comments like S2I images.

File: install/disconnected_install.adoc [hunk 4]
```diff
@@ -578,247 +610,44 @@ name=rhel-7-server-ansible-2.4-rpms
...
-
-.. Verify that all the image streams use the correct tags:
-+
-[source, bash]
-----
-$ oc get imagestreams -n openshift
-NAME                                 DOCKER REPO                                                      TAGS                                     UPDATED
-jboss-webserver30-tomcat7-openshift  $REGISTRY/jboss-webserver-3/webserver30-jboss-tomcat7-openshift  1.1,1.1-2,1.1-6 + 2 more...              2 weeks ago
-...
+orge_url=registry.example.com/testing/ocp3/ose-${component}:${version}
```
[the remark this answers is not in the document]
Comment (kalexand-rh) [reply install/disconnected_install.adoc]: Thank you! I made a new PR to fix this in https://github.com/openshift/openshift-docs/pull/12179

**Pull request merged**